test/collectionUtils/ModifyElementTypeTransformer.test.ts

```diff
@@ -0,0 +1,47 @@
+import EntityDefinitionBuilder from "../model/dataTypes/EntityDefinitionBuilder";
+import EntityDefinition from "../../src/model/dataTypes/EntityDefinition";
+import DataType from "../../src/model/dataTypes/DataType";
+import ModifyElementTypeTransformer from "../../src/collectionUtils/ModifyElementTypeTransformer";
+import EntityImports from "../../src/model/dataTypes/EntityImports";
+
+describe("ModifyElementTypeTransformer", () => {
+  const transformer = new ModifyElementTypeTransformer();
+  let entityBuilder: EntityDefinitionBuilder;
+  let otherEntity: EntityDefinition;
+  let elementEntity: EntityDefinition;
+  let elementType: DataType;
+  let otherType: DataType;
+
+  beforeEach(() => {
```

Comment:
Codacy found an issue: [Unsafe call of an any typed value.](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5966907)